community.hachyderm.io/content/en/blog/posts/2024-01-moderator-minute/index.md

```diff
@@ -0,0 +1,103 @@
+
+What this means:
+- You will still be able to follow people from Threads when they federate those users
+- You will not see Threads posts on your timelines unless you follow the user
+- You will be required to approve all followers from Threads
+- You will be required to accept a prompt before viewing a Threads user’s account unless you are following them
+
+### Next Steps
+
+As Threads continues to implement their integration with ActivityPub and the Fediverse at large, we will watch how those users integrate with our community and how their service interacts with our servers. If you would like to learn more about our criteria for how Hachyderm handles federating with other instances, please review our A Minute from the Moderators - July Edition where we list out our criteria.
+
+##### Very, very quick Q&A on Threads
+
+**Q: What happens if there is a problematic account on Threads that is federating?**
+
+A: They will be blocked by our existing moderation policies.
+
+**Q: What happens if there are multiple problematic accounts on Threads that are federating?**
+
+A: Depending on the volume, we will either block all those individual accounts or Threads as a whole.
+
+**Q: I am concerned about anti-trans and other hate content that may come from Threads.**
+
+A: We are too! And we continue to monitor the instance for any of these behaviors. If and when that type of content appears, we will block it the way we do with any other instance.
+
+**Q: Why are you giving Threads “a chance”? We know about the parent company.**
+
+A: We’re not giving Threads “a chance” so much as meeting the current situation where it’s at with plans to continue to adapt as the situation changes. Since there are 9 users, we’re treating this instance like a 9 user instance. In cases with instances that are ~50 users or less, unless the instance is cohesive (e.g. examplenazi.com where they all espouse that ideology), we moderate individual accounts. This means that regardless of the status of Threads as a whole or the number of federating accounts, accounts like Libs of Tik Tok would never federate with Hachyderm for being in violation of our federating policy.
+
+## Community Collaboration
+
+### Community Votes
+In the approximately 15 months since Hachyderm scaled, we have limited or suspended federation with 19 instances per month for a total of 285 instances. These instances in large part have been banned in order to pre-emptively protect our users. In almost every case, we know that these instances go against everything we stand for as a group. However, in a few of those cases, it’s more nuanced to determine if/when we should block an instance and how we should communicate those changes.
+
+For example, when we blocked exampleillegal.com for illicit materials. We didn’t announce the change and we expect that none of our users will take issue with us blocking that material.

```

Comment:
For example, when we block sites like exampleillegal.com for illicit content.

(Need to change the phrasing to match that we're not using an actual example)